### ISSUE TITLE
Multiple Sites - PHP Fatal Error: Cannot declare class because name is already in use

### ISSUE THREAD

**Issue by kurtaschliman** [laradock/laradock#1880]
### Info:
- Docker version (`$ docker --version`): 18.06.1-ce, build e68fc7a
- Laradock commit (`$ git rev-parse HEAD`): 0ed5487ed7204aff6897db8de14e1e79469280fd
- System info (Mac, PC, Linux): PC
- System info disto/version: Windows 10

### Issue:
I am working with multiple installations of WordPress using Laradock. When I visit one site (we'll call it Test1) for the first time, everything looks great. Test2 is an exact clone of Test1, but with a different database and separate file path (/var/www/Test1 & /var/www/Test2). When I try to load Test2, it throws a 500 Error:

```
Fatal error: Cannot declare class wpai_acf_add_on\acf\fields\FieldGoogleMap, because the name is already in use in /var/www/test1.local/wp-content/plugins/wpai-acf-add-on/libraries/acf-fields/fields/fieldGoogleMap.php on line 0
```

I've been trying to work around this for a couple of days now. It feels like a misconfiguration with my local environment, so I essentially nuked everything (Fresh Laradock, Reinstalled Docker, wiped all data volumes / networks / images, rebuilt everything, etc) but cannot get around this fatal error.

This feels like somehow the PHP process loading Test2 already has Test1 loaded. Opcache is off and no other object caching is involved, afaik. 

Looking for some advice on what to try next as this is causing a headache and I'm out of my own brain power to debug further. I realize there are some specifics that might be necessary that I am leaving out, so ask questions and I can provide further information as best I can. Again, just looking for direction, and if it's some way that I am configuring this thing, maybe it will help someone else down the road. Thanks in advance.
_____

### Relevant Code:

Full stacktrace of the error:
```
Fatal error: Cannot declare class wpai_acf_add_on\acf\fields\FieldGoogleMap, because the name is already in use in /var/www/test1.local/wp-content/plugins/wpai-acf-add-on/libraries/acf-fields/fields/fieldGoogleMap.php on line 0
--
1 | 0.0000 | 404104 | {main}( ) | .../plugins.php:0
2 | 0.3298 | 3210864 | plugin_sandbox_scrape( ) | .../plugins.php:178
3 | 0.3299 | 3213768 | include( '/var/www/test2.local/wp-content/plugins/wpai-acf-add-on/wpai-acf-add-on.php' ) | .../plugin.php:1897
4 | 0.3301 | 3214120 | PMAI_Plugin::getInstance( ) | .../wpai-acf-add-on.php:534
5 | 0.3301 | 3214360 | PMAI_Plugin->__construct( ) | .../wpai-acf-add-on.php:100
6 | 0.4029 | 3360008 | require_once( '/var/www/test2.local/wp-content/plugins/wpai-acf-add-on/libraries/acf-fields/fields/fieldGoogleMapExtended.php' ) | .../wpai-acf-add-on.php:193
```

Here is the beginning of the class that won't load in separate sites:
```
<?php

namespace wpai_acf_add_on\acf\fields;

use wpai_acf_add_on\acf\ACFService;

require_once PMAI_ROOT_DIR . '/libraries/acf-fields/base/baseGoogleMap.php';

/**
 * Class FieldGoogleMap
 * @package wpai_acf_add_on\acf\fields
 */
class FieldGoogleMap extends BaseGoogleMap {

...
```

**Comment by yayMark** [comment]
Did you find a solution to this? I get this when I try to install the WP All Import ACF addon for the first time.

**Comment by andruxnet** [comment]
I have this problem as well, any solution?

**Comment by andruxnet** [comment]
Found the solution here: https://github.com/laradock/laradock/issues/2392#issuecomment-561616264

TLDR;
Disable PHP OPCache in your .env file. ie PHP_FPM_INSTALL_OPCACHE=false
